src/devices/Device-Index.md

```diff
@@ -80,7 +80,9 @@
 * [Sense HAT](SenseHat/README.md)
 * [SensorHub - Environmental sensor](SensorHub/README.md)
 * [Servo Motor](ServoMotor/README.md)
+* [SCD4x - CO<sub>2</sub>, Temperature & Humidity Sensor](Scd4x/README.md)
```

Comment:
No need for you to update this file manually. We have a tool that we run in a ~monthly cadence, which checks all the device bindings we have and updates this list automatically.